README.rst

```diff
@@ -943,6 +943,37 @@ To use point ``STATICFILES_STORAGE`` to ``openwisp_utils.storage.CompressStaticF
 
     STATICFILES_STORAGE = 'openwisp_utils.storage.CompressStaticFilesStorage'
 
+``openwisp_utils.admin_theme.email.send_email``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This function let you send customised mail using the template and logo that can be customised using 
+`OPENWISP_EMAIL_TEMPLATE <#openwisp_email_template>`_ and `OPENWISP_EMAIL_LOGO <#openwisp_email_logo>`_ respectively.
+
+You can use `OPENWISP_HTML_EMAIL <#openwisp_html_email>`_ option to choose whether html template should be used or plain text should be sent.
+
+
+**Syntax:**
+
+.. code-block:: python
+
+    send_email(subject, body, recipients)
+
++--------------------+----------------------------------------------------------------------------------+
+| **Parameter**      | **Description**                                                                  |
++--------------------+----------------------------------------------------------------------------------+
+| ``subject``        | (``str``) The subject of the email template.                                     |
++--------------------+----------------------------------------------------------------------------------+
+| ``body``           | (``str``) The body of the email template.                                        |
++--------------------+----------------------------------------------------------------------------------+
+| ``recipients``     | (``list``) The list of recipients you want to send the mail to.                  |
++--------------------+----------------------------------------------------------------------------------+
+| ``extra_context``  | **optional** (``dict``) Extra context you want to pass.                          |
++--------------------+----------------------------------------------------------------------------------+
+
+``call_to_action_text`` and ``call_to_action_url`` can be passed in ``extra_context`` to show a call to action button.
```

Comment:
Can you please include this info in the table cell describing extra_context? See how it's done in  `Adding a custom link`.

I'd write it as something like: 

```
The dictionary keys ``call_to_action_text`` and ``call_to_action_url`` can be passed to show a call to action button.
```